scripts/tool_shed/build_ts_whoosh_index.py

```diff
@@ -6,7 +6,9 @@
 
 Make sure you adjusted your config to:
  * turn on searching via toolshed_search_on
- * specify toolshed_whoosh_index_dir where the indexes will be placed
+ * specify whoosh_index_dir where the indexes will be placed
+
+Also make sure that GALAXY_EGGS_PATH variable is properly set.
```

Comment:
Yeah it runs from the box, but if you set your GALAXY_EGGS_PATH for galaxy you have to set it for this script too.